src/Cosi-BLS/block.lisp

```diff
@@ -6,43 +6,48 @@
 
 
                                                                                 ;
-(defclass block ()
+(defclass eblock ()
```

Comment:
Note need to sync with `*names-of-block-slots-to-serialize*` in a comment.  Possibly place nearby in source.